backend/src/main/kotlin/ch/yvu/teststore/run/overview/RunOverviewService.kt

```diff
@@ -36,11 +40,11 @@ open class RunOverviewService @Autowired constructor(
         run.testSuite?.let { testSuite ->
             run.time?.let { time ->
                 run.id?.let { runId ->
-                        val diff = resultDiffService.findDiff(null, runId)
-                        val numberPassed = diff[ResultDiffService.DiffCategory.NEW_PASSED]?.size ?: 0
-                        val numberFailed = diff[ResultDiffService.DiffCategory.NEW_FAILED]?.size ?: 0
+                    val diff = resultDiffService.findDiff(null, runId)
```

Comment:
For PRs in the future: A good practice to make reviews easier, is to separate "code reformating"-commits from commits with actual behavior changes.